include/pmacc/PMaccConfig.cmake

```diff
@@ -346,25 +334,10 @@ elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DBOOST_NO_CXX11_SMART_PTR")
 endif()
 
-# Boost 1.64.0 is broken with CUDA 8.0 (nvcc) and C++11
-#   https://github.com/ComputationalRadiationPhysics/picongpu/issues/2048
-#   fixed in CUDA 9.0 (ticket 1928813)
-if( ("${ALPAKA_CUDA_COMPILER}" STREQUAL "nvcc") AND
-    (Boost_VERSION EQUAL 106400) AND
-    (CUDA_VERSION VERSION_EQUAL 8.0) )
```

Comment:
TODO: hm, we don't really check the CUDA version here since we take whatever alpaka gives us....

Reply:
Discussed offline with @ax3l : We will drop the CUDA 8 support fully. IN that case it is fine to remove the workaround.

Reply:
We should still add a `message(ERROR ...)` on the CUDA version during CMake time :)